src/mixins/wallet.js

```diff
@@ -32,6 +32,8 @@ export default {
       'walletIsMatchedSession',
       'walletIsLoggingIn',
       'loginAddress',
+      'getFolloweeEvents',
+      'walletIsFetchingFolloweeEvents',
```

Comment:
If these computed props are only used in certain components, we might not want to add to wallet mixins, as the computed props update, those props will trigger unnecessary rendering